src/solvers/dg/dg_1d.jl

```diff
@@ -6,11 +6,11 @@
 # It constructs the basic `cache` used throughout the simulation to compute
 # the RHS etc.
 function create_cache(mesh::TreeMesh{1}, equations::AbstractEquations{1},
-                      dg::DG, RealT)
+                      dg::DG, RealT, uEltype)
```

Comment:
Just a general question (again): Should we define defaults for `RealT` and `uEltype` (e.g., `real(dg)`) or is there no sensible default?

Reply:
We could define default values. My reason for not doing this was that it's easier to check whether some call-sites need to be updated. Since its mostly used internally, I thought it would be better to alert developers explicitly when changing the signature (since the call-sites will probably need some updates as well).

Reply:
Agreed!